google.json

```diff
@@ -509,7 +514,7 @@
     {
         "id": "com.google.android.apps.searchlite",
         "label": "Google Go",
-        "description": "The Google app made for low-RAM devices.",
+        "description": "The Google search app made for low-RAM devices.\n",
```

Comment:
Description should not end with `\n`. They are present in the UAD project for some reason unknown to me. If you can't, I'll do it after merging the PR.

Reply:
Noted. That's a lot of work if I clean them ALL manually, please change from your side.